predicators/envs/blocks.py

```diff
@@ -28,7 +28,7 @@ class BlocksEnv(BaseEnv):
     """Blocks domain."""
     # Parameters that aren't important enough to need to clog up settings.py
     table_height: ClassVar[float] = 0.2
-    block_size: ClassVar[float] = 0.045
+    block_size: ClassVar[float] = 0.0505
```

Comment:
The true block size is `0.05` but we added the buffer of 5mm to avoid the collision reflex on the robot. 

Should we just continue to use this buffer or add logic in the controller to place blocks marginally higher than the other ones?

Reply:
I'm gonna leave it as is for now because dropping the blocks from higher in simulation may lead to a little bouncing